### ISSUE TITLE
Node can be passed to shape function

### ISSUE THREAD

**Issue by colinmeinke** [colinmeinke/wilderness#67]
Moving this from [wilderness-core](https://github.com/colinmeinke/wilderness-core).

```js
const s1 = shape(el)
const s2 = shape([ el, { fill: 'yellow' } ])
```

Or in conjunction with https://github.com/colinmeinke/wilderness-core/issues/2

```js
const s = shape([ el, { fill: 'yellow' } ], { replace: el })
```

Requires a function to create a Plain Shape Object from a node. Maybe abstract to svg-points lib.

**Comment by colinmeinke** [comment]
Final API for this is:

```js
const s = shape({ el, fill: 'yellow' })
```

**Comment by colinmeinke** [comment]
Closed in next 58bff6c0daca0e284dfa7779a93752b08ba17615
